packages/core/content-releases/server/src/controllers/validation/release-action.ts

```diff
@@ -4,8 +4,9 @@ const RELEASE_ACTION_SCHEMA = yup.object().shape({
   entry: yup
     .object()
     .shape({
-      id: yup.strapiID().required(),
+      documentId: yup.strapiID(),
```

Comment:
It's not required?

Reply:
Is not required because with SingleTypes you can create a ReleaseAction only with the ContentTypeUID and the locale